ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-master__edge.yaml

```diff
@@ -11,6 +11,14 @@ base_images:
     name: ocm-2.7
     namespace: edge-infrastructure
     tag: assisted-service
+  assisted-service-index:
+    name: assisted-service-index
+    namespace: edge-infrastructure
+    tag: latest
```

Comment:
another question about this index, maybe more relevant to the previous PR of supplemental images
don't we have a similar mechanism for updating this index?
https://github.com/openshift/release/blob/0896191a7c5ec03f0bbb9038bc1fe8ac14f887a2/ci-operator/step-registry/assisted/baremetal/operator/catalog-publish/assisted-baremetal-operator-catalog-publish-ref.yaml#L36-L43

Reply:
I'm not sure to understand your question, what do you mean by "don't we have a similar mechanism for updating this index" ? The `assisted-baremetal-operator-catalog-publish` step publishes the image directly to quay.

Reply:
So ``assisted-baremetal-operator-catalog-publish`` publishes index to ``quay.io/edge-infrastructure/assisted-service-index:latest``, and the supplemental-images mirroring mirrors it to ``registry.ci.openshift.org/edge-infrastructure/assisted-service-index:latest``
In addition, here we're promoting the index to namespace ``edge-infrastructure`` and name ``assisted-service-index:latest`` (at least from what I understand) and promotion is automatically available at ``registry.ci.openshift.org``. How does those two paths of updating ``registry.ci.openshift.org/edge-infrastructure/assisted-service-index:latest`` play along?

Reply:
It's a bit technical, so if I'm just missing some detail here let me know

Reply:
the broken index we promote from `ci-index` is accessible from `registry.ci.openshift.org/edge-infrastructure/ocm-2.7:assisted-service-operator-index`: https://github.com/openshift/release/blob/master/ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml#L160.

Their names do not collide.

Reply:
missed that
thanks!

Reply:
no problem !